Configuration/Geometry/README.md

```diff
@@ -56,5 +62,7 @@ Several detector combinations have been generated:
 * D54 = T19+C9+M4+I10+O4+F2
 * D55 = T19+C9+M5+I10+O4+F2
 * D56 = T20+C9+M4+I10+O4+F2
+* D57 = T19+C11+M6+I11+O5+F4
+* D58 = T19+C12+M6+I11+O5+F5
```

Comment:
Why Tracker T19 is taken here in these new scenarios? That's an experimental version with 3D pixels in TBPX1,2 and TPFX Ring1, which is still largely unvalidared. The current baseline for planar sensors is T17 to be replaced by T20 once that's validated.  @emiglior @ghugo83.

Reply:
This is a good point, in principle these development geometries should stick with T17 (or even T15, which is still the TDR baseline).

Reply:
Using T17 now

Reply:
Yes absolutely, please use T17.